ts/features/idpay/initiative/configuration/screens/InstrumentsEnrollmentScreen.tsx

```diff
@@ -299,9 +301,17 @@ const InstrumentsEnrollmentScreen = () => {
                 ))}
               </NBList>
               <VSpacer />
-              <NBText>
-                {I18n.t("idpay.configuration.instruments.footer")}
-              </NBText>
+              <View style={styles.bottomSection}>
+                <IconNav name="wallet" color="bluegrey" />
+                <HSpacer size={8} />
+                <LabelSmall
+                  color="bluegrey"
+                  weight="Regular"
+                  style={IOStyles.flex} // required for correct wrapping
+                >
+                  {I18n.t("idpay.configuration.instruments.footer")}
+                </LabelSmall>
+              </View>
```

Comment:
Please use `AdviceComponent` for the banner

Reply:
not possible: `AdviceComponent` runs with `IconFont`, which does not have the desired icon and has some funny centering for our purposes. awaiting design for new component